src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -80,23 +84,30 @@ public class SwerveSubsystem extends SubsystemBase {
   private Pose2d pose = new Pose2d();
   private Rotation2d lastGyroRotation = new Rotation2d();
 
-  private final VisionIO visionIO;
-  // private final VisionIOInputsAutoLogged visionInputs = new VisionIOInputsAutoLogged();
-  private final VisionIOInputsLogged visionInputs = new VisionIOInputsLogged();
+  private final Vision[] cameras;
   private AprilTagFieldLayout fieldTags;
   public SwerveDrivePoseEstimator estimator;
   Vector<N3> odoStdDevs = VecBuilder.fill(0.3, 0.3, 0.01);
   Vector<N3> visStdDevs = VecBuilder.fill(1.3, 1.3, 3.3);
   private SwerveModulePosition[] modulePositions = new SwerveModulePosition[4];
-
-  public SwerveSubsystem(GyroIO gyroIO, VisionIO visionIO, ModuleIO... moduleIOs) {
+  public static final Transform3d leftCamToRobot = new Transform3d(); // TODO find
+  public static final Transform3d rightCamToRobot = new Transform3d(); // TODO find
```

Comment:
these transforms shouldnt need to be in their own variables and can just be defined in the constant